pom.xml

```diff
@@ -14,6 +14,11 @@
   <name>Jenkins SSH Slaves plugin</name>
   <url>http://wiki.jenkins-ci.org/display/JENKINS/SSH+Slaves+plugin</url>
 
+  <properties>
+    <findbugs-maven-plugin.version>3.0.1</findbugs-maven-plugin.version>
```

Comment:
I *hate* people using properties for a dependency/plugin version that is used once and only once in the pom. Using properties for versions is really not a good plan unless you expressly know exactly what you are doing and why you are doing it.